### ISSUE TITLE
WPF AutomationProperties.LiveSetting="Polite" not triggering live region updates

### ISSUE THREAD

**Issue by GeoffYin** [dotnet/wpf#11125]
### Description

I have an AccessText element with text that updates every second. I hope that each text update doesn't interrupt the previous reading by Narrator. However, when I set the property AutomationProperties.LiveSetting = "Polite", it still doesn't work. Narrator still interrupts the previous reading and starts reading the next text update. the version of dotnet framwork is 4.8

### Reproduction Steps

1. Add an AccessText that updates once every second, and set the property AutomationProperties.LiveSetting = "Polite".
2. Turn on Narrator scan mode.
3. Use the arrow keys to make Narrator focus on the AccessText.

### Expected behavior

The expected result is that after setting the AutomationProperties.LiveSetting = "Polite" property, Narrator should not interrupt the previous text reading, just like the aria - live="polite" property in HTML.

### Actual behavior

Narrator interrupts the previous reading and starts reading the newly updated text even before finishing the previous one.

**Comment by lindexi** [comment]
@GeoffYin Do you set `Switch.UseLegacyAccessibilityFeatures`? 

https://github.com/dotnet/wpf/blob/ee7cbb82652173e3a9f4e8feec155eb58eed5f4f/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Automation/Peers/AutomationPeer.cs#L2424-L2427

https://github.com/dotnet/wpf/blob/ee7cbb82652173e3a9f4e8feec155eb58eed5f4f/src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/AccessibilitySwitches.cs#L46-L65

**Comment by GeoffYin** [comment]
@lindexi no, I didn't set.

**Comment by lindexi** [comment]
@GeoffYin It seems that WPF already pass the LiveSetting property to UIA.

**Comment by GeoffYin** [comment]
@lindexi  but it still doesn't work, Narrator still interrupt the previous reading and start reading the next text update. Is this a problem with Narrator?

**Comment by lindexi** [comment]
@GeoffYin Sorry, I lack methods for further investigation. I'm not sure if only Polite LiveSetting  is enough.
